run.py

```diff
@@ -1,26 +1,35 @@
+from fileinput import filename
 import os,time
 from datetime import datetime
 import shutil
 
-while(True):
-    backup_frequency = 2 #os.environ['BACKUP_FREQUENCY', 60]
-    backup_age = 1 #os.environ['BACKUP_AGE',1]
-    source = "/home/kaz/DockerFiles/saves"
-    destination= "/home/kaz/DockerFiles/backups"
+def main():
+    while(True):
+        backup_frequency = os.environ['BACKUP_FREQUENCY', 60]
+        backup_data()
+        delete_old_files()  
+        time.sleep(backup_frequency*60)
+
+def backup_data():
+
     fileName = datetime.now().strftime("%Y_%m_%d-%H_%M_%S_%p")
+    source = "/saves"
+    destination= "/backups"
 
-    shutil.make_archive(fileName, 'zip', source)
-        
-    files_list = os.listdir(destination)
-    current_time = time.time()
-    for backup_files in files_list:
-        file_path = os.path.join(destination, backup_files)
-        if os.path.isfile(file_path):
-            if (current_time - os.stat(file_path).st_birthtime) > backup_age * 86400:
-                os.remove(file_path)    
-            
-        print("test")
+    shutil.make_archive(base_name=(os.path.join(destination,fileName)), format="zip", root_dir=source)
 
-    time.sleep(backup_frequency*60)
+def delete_old_files():
```

Comment:
Pass in files here to make this function easier to test. Maybe you should have a function that takes a list of files and returns old ones.